envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -53,6 +53,11 @@ def cves_class(self) -> "abstract.ADependencyCVEs":
         """CVEs class."""
         raise NotImplementedError
 
+    @cached_property
+    def dep_ids(self) -> Tuple[str, ...]:
```

Comment:
this bit is actually required for release-issues - i can shift if it if preferable